--- origsrc/kdeedu-4.5.5/cmake/modules/FindBoostPython.cmake	2010-06-08 05:10:20.000000000 -0500
+++ src/kdeedu-4.5.5/cmake/modules/FindBoostPython.cmake	2011-01-14 01:23:34.726869800 -0600
@@ -25,7 +25,9 @@
 set(BOOST_PYTHON_LIBS)
 
 # handy arrays
+if(NOT PYTHON_VERSIONS)
 set(PYTHON_VERSIONS "python;python2.7;python2.6;python2.5;python2.4;python2.3;python2.2")
+endif()
 
 # 1st: check for boost/shared_ptr.hpp
 check_include_file_cxx(boost/shared_ptr.hpp HAVE_BOOST_SHARED_PTR_HPP)
